cypress_test/integration_tests/desktop/writer/sidebar_spec.js

```diff
@@ -65,4 +65,29 @@ describe(['tagdesktop'], 'Sidebar tests', function() {
 		cy.cGet('#expand_other').should('exist');
 	});
 
+	it('Focus on the first focusable child', function() {
+		cy.cGet('#sidebar-dock-wrapper')
```

Comment:
nice function which could be useful in many contexts, please extract to separate helper method like `containsFocusedElement(container: Element, doesContain: boolean = true)` and put in `common/helper.js`

we can reuse it here as we have already 2 places :)

Reply:
Yes, it could be useful.